packages/salesforcedx-vscode-aura/.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
We already have a top level `.prettierrc` that applies to the rest of the monorepo. Is there a specific reason a different config is needed ?